src/main/java/model/LadderValidator.java

```diff
@@ -0,0 +1,32 @@
+package model;
+
+import java.util.*;
+import java.util.stream.*;
+
+public class LadderValidator {
+    private static final Random RANDOM = new Random();
+
+    public static void validate(List<Line> lines, int width) {
+        IntStream.rangeClosed(0, width)
+                .forEach(i -> validateColumn(lines, i, width));
+    }
+
+    private static void validateColumn(List<Line> lines, int col, int width) {
+        boolean emptyColumn = lines.stream().noneMatch(line -> hasBridgeAt(line, col, width));
```

Comment:
```suggestion
        boolean emptyColumn = lines.stream()
                .noneMatch(line -> hasBridgeAt(line, col, width));
```
메소드 체이닝을 해야 할 때는 도트 연산자마다 개행을 해주는 편이 낫다고 생각합니다. 상화의 생각은 어떤가요? 🤔